src/main/java/at/hannibal2/skyhanni/features/mining/TunnelsMaps.kt

```diff
@@ -180,7 +182,14 @@ object TunnelsMaps {
         }.toMap()
         if (config.autoCommission) {
             clickTranslate.values.firstOrNull()?.let {
+                isCommission = true
                 setActiveAndGoal(it)
+            } ?: run {
+                if (isCommission) {
+                    active = ""
+                    goal = null
```

Comment:
clear path would be more correct here. Since the path does not need to exist without a goal. Normally goal setting clears the old path. But when the goal is null it returns early. Meaning the path that the path should be explicitly cleared.